src/rules/selector-no-redundant-nesting-selector/__tests__/index.js

```diff
@@ -345,6 +345,50 @@ testRule(rule, {
       message: messages.rejected,
       description:
         "when an ampersand is used in a comma sequence followed by a class"
+    },
+    {
+      code: `
+      @theme: ~'dark';
+      p {
+        & when (@theme = dark) {}
+        & when not (@theme = dark) {}
+      }
+    `,
+      line: 4,
```

Comment:
Also add the `column` property here.

Reply:
@kaysonwu can you still fix this one? The `column` property is missing from the test.

Reply:
I've fixed it, but it's not synced. I'll check it

Reply:
Now, it seems to be synchronized, I added the `color` property